bin/dredd

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env node
+
```

Comment:
This is awesome! Would it be possible to keep it here in this file and enable the functionality by exporting some ENV variable? I would love to have one file. I'm afraid diverging of those two might happen.